src/main/resources/application.properties

```diff
@@ -0,0 +1,24 @@
+# ===============================
+# DATABASE
+# ===============================
+ 
+spring.datasource.driver-class-name=com.mysql.jdbc.Driver
+spring.datasource.url=jdbc:mysql://localhost:3306/hiddenroletogo
+spring.datasource.username=root
+spring.datasource.password=passw0rd
+ 
+# ===============================
+# JPA / HIBERNATE
+# ===============================
+ 
+spring.jpa.show-sql=true
+spring.jpa.hibernate.ddl-auto=update
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL5InnoDBDialect
+
+#
```

Comment:
nit: can you add the ==== around RABBIT as well? and include that it's the server/host, equivalent to the 'DB, JPA' qualifiers we have for the others??

Reply:
Thanks! Forgot to look back at the props file after all the hacky stuff I was doing to get it working.